frappe/__init__.py

```diff
@@ -2068,25 +2068,36 @@ def logger(
 	)
 
 
-def log_error(message=None, title=_("Error")):
+def log_error(title=None, message=None, reference_doctype=None, reference_name=None):
```

Comment:
Although this order of parameters makes more sense, changing the order in this function is dangerous for dependent apps as well as instances in Frappe and ERPNext. Most of the instances don't send keyword arguments while calling this function assuming the message is the first argument followed by the title like:

`frappe.log_error(frappe.get_traceback(), "Razorpay Payment not authorized")`

This leads to Truncation errors and the error log isn't captured.